Packs/Armis/Layouts/layoutscontainer-Armis_Alert.json

```diff
@@ -0,0 +1,497 @@
+{
+    "group": "incident",
+    "id": "Armis Alert",
+    "name": "Armis Alert - Layout",
```

Comment:
Please change the name to match the id here and make sure to change in all relevant places